spec.emu

```diff
@@ -69,6 +69,34 @@ contributors: Ujjwal Sharma, Younies Mahmoud
 
     <emu-note type=editor>The [[Weeks]] field is still under discussion in Temporal. For more, check out the <a href="https://github.com/tc39/proposal-temporal/issues/532">Github thread</a> on the Temporal issue tracker.</emu-note>
 
+    <emu-clause id="sec-actuallargestunit" aoid="ActualLargestUnit">
+      <h1>ActualLargestUnit ( _largestUnit_, _duration_ )</h1>
+
+      <p>The ActualLargestUnit abstract operation is called with the arguments _largestUnit_ (which must be a string) and _duration_ (which must be a Record with the fields enumerated in <emu-xref href="#table-duration-components"></emu-xref>), and returns the largest non-zero unit on _duration_ smaller than _largestUnit_. The following steps are taken:</p>
+
+      <emu-alg>
+        1. For each row in <emu-xref href="#table-duration-components"></emu-xref>, starting from _largestUnit_, in table order, do
+          1. Let _field_ be the Property value.
+          1. Let _val_ be _duration_.[[&lt;_field_&gt;]].
+          1. If _val_ is not `undefined` and not `0`, then
+            1. Return _field_.
+      </emu-alg>
```

Comment:
Don't you need
```suggestion
            1. Return _field_.
        1. Return `"nanoseconds"`.
      </emu-alg>
```
? (backticks need to become asterisks at some point too.)

Reply:
I had that but removed it considering that we should weed out zero durations earlier... Just realized that we aren't doing that yet. Would you like me to do this instead of short-circuiting in case of zero durations?

Reply:
If you're sure it's not needed, add an assertion and make the paragraph above say something like `_duration_ (which must be a Record with the fields enumerated in <emu-xref href="#table-duration-components"></emu-xref>, where not all the fields are zero)`.